pkg/generators/openapi.go

```diff
@@ -416,13 +423,13 @@ func (g openAPITypeWriter) generateMemberExtensions(m *types.Member, parent *typ
 			klog.V(2).Infof("%s %s\n", errorPrefix, e)
 		}
 	}
-	g.emitExtensions(extensions)
+	g.emitExtensions(extensions, nil)
 	return nil
 }
 
-func (g openAPITypeWriter) emitExtensions(extensions []extension) {
+func (g openAPITypeWriter) emitExtensions(extensions []extension, unions []union) {
```

Comment:
Is there a reason why unions isn't just a part of extensions?

Reply:
Yeah, I would have to completely change the way extensions are built (they're meant for list of scalars) and I didn't want to make the type worse by adding a bunch of unrelated things.

Reply:
sounds good